packages/utilities/package.json

```diff
@@ -36,7 +36,6 @@
     "typescript": "^4.3.2"
   },
   "dependencies": {
-    "dayjs": "^1.10.5",
-    "timezone-support": "^2.0.2"
+    "dayjs": "^1.10.5"
```

Comment:
Update the yarn.lock file as well after running yarn install.

Reply:
Updated